clients-src/modules/ROOT/pages/clients.adoc

```diff
@@ -6,125 +6,76 @@
 
 == Overview
 
-A TypeDB Client is any software that can connect to a TypeDB.
-It is meant to be used for the purpose of managing databases and performing operations (querying) with a
-database schema and data on the TypeDB server.
+Using a TypeDB Client is the primary way to interact with a TypeDB server. These include standalone applications
+as well as lower-level libraries called Drivers.
 
-TypeDB uses its own https://github.com/vaticle/typedb-protocol[TypeDB Client RPC,window=_blank] protocol via `gRPC`
-interface to accept connections from Clients. To connect we can use xref:studio.adoc[TypeDB Studio],
-xref:console.adoc[TypeDB Console], or we can build our own application (Client).
-
-A TypeDB Client uses `gRPC` to connect with a TypeDB server and provides more user-friendly interface: API, GUI or CLI.
-
-To build an application capable of connecting to TypeDB via *TypeDB Client RPC* protocol we can use one of the existing
-libraries implementing it, called <<_typedb_drivers,TypeDB Drivers>>, or we can implement it ourselves as a
-xref:new-driver.adoc[new driver].
+All Clients use a network `grpc` https://github.com/vaticle/typedb-protocol[protocol,window=_blank] under the hood
+to communicate with a TypeDB server.
 
 image::client-server-comms.png[Client types and server interaction]
 
 //#todo Update terminology on image
 
-Here is a list of existing *TypeDB Clients*:
-////
-.TypeDB Clients
-[cols="^1,^3,^3",options="header"]
-|===
-| Client type | Built by Vaticle | Built by community
-
-| Applications
-| TypeDB Studio,
-TypeDB Console
-| TypeDB Loader
-
-| Drivers
-| Java, Python, Node.js, Rust
-| Julia, Go, Haskell, C#
-|===
-////
-
-* xref:studio.adoc[TypeDB Studio]
-* xref:console.adoc[TypeDB Console]
-* <<_typedb_drivers,TypeDB Drivers>> -- libraries to connect to a TypeDB server with an application. There are:
-  ** xref:java/java-overview.adoc[Java]
-  ** xref:python/python-overview.adoc[Python]
-  ** xref:node-js/node-js-overview.adoc[Node.js]
-  ** Rust (https://github.com/vaticle/typedb-client-rust[Under development,window=_blank])
-
-If there is no suitable client in the list, check the xref:other-languages.adoc[Community drivers],
-or consider xref:new-driver.adoc[creating a new client].
-
-There is also a community driven Jupyter notebook plugin to work with TypeDB:
-https://pypi.org/project/typedb-jupyter/[TypeDB Jupyter connector,window=_blank].
 
+//#todo putting protocol versioning as a 'WARNING' is off-putting - we can frame it as a good thing!
```

Comment:
Note to ourselves: we should frame decisions we take in a positive light. Protocol versioning _can_ break your connection if you upgrade, but that an important validation, rather than a dangerous warning